### PR TITLE
No issue - Mainly fixes in documentation

### PR DESCRIPTION
**Pull request by Lonnytunes**
I have a doubt about the precision of the `Core` directory when loading default anonymizers or mentioning them in documentation. If it is not a good idea, I let you remove it :wink:

### CODE REVIEW

File: config/db_tools.standalone.complete.sample.yaml [hunk 1]
```diff
@@ -33,12 +33,12 @@
...
 #restore_timeout: 2400 # default 1800
 
-# List here tables (you don't want in your backups.
+# List here tables you don't want in your backups.
 # If you have more than one connection, it is strongly advised to configure
 # this for each connection instead.
 #backup_excluded_tables: ['table1', 'table2']
 
-# Specify here paths to backup and restore binaries and command line options.
+# Specify here paths to back up and restore binaries and command line options.
```
Comment (pounard) [inline config/db_tools.standalone.complete.sample.yaml:41]: You should probably keep "backup" here.

File: config/db_tools.standalone.complete.sample.yaml [hunk 2]
```diff
@@ -54,7 +54,7 @@
 # Connection name will be "default" when configured this way.
 connections: "pgsql://username:password@hostname:port?version=16.0&other_option=..."
 
-# Alternatively, you are allowed to have more than one connections, case
+# Alternatively, you are allowed to have more than one connection, case
```
Comment (pounard) [inline config/db_tools.standalone.complete.sample.yaml:57]: "More" than one, shouldn't be plural "connections" here ?

Comment (pounard) [reply config/db_tools.standalone.complete.sample.yaml:57]: Or that may be a frenchism, I'm not sure about english grammar here.

File: config/packages/db_tools.yaml [hunk 3]
```diff
@@ -29,12 +30,12 @@ db_tools:
...
     #restore_timeout: 2400 # default 1800
 
-    # List here tables (you don't want in your backups.
+    # List here tables you don't want in your backups.
     # If you have more than one connection, it is strongly advised to configure
     # this for each connection instead.
     #backup_excluded_tables: ['table1', 'table2']
 
-    # Specify here paths to backup and restore binaries and command line options.
+    # Specify here paths to back up and restore binaries and command line options.
```
Comment (pounard) [inline config/packages/db_tools.yaml:38]: Same as upper, I would keep "backup" as a single word?

File: docs/content/backup_restore.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ You can specify the behavior of the command with some options detailed below.
 
 <div class="standalone">
 
-By default, the command will backup the database from the default connection.
+By default, the command will back up the database from the default connection.
```
Comment (pounard) [inline docs/content/backup_restore.md:39]: Same.

File: docs/content/backup_restore.md [hunk 5]
```diff
@@ -48,7 +48,7 @@ vendor/bin/db-tools backup --connection other_connection_name
 </div>
 <div class="symfony">
 
-By default, the command will backup the database from the default DBAL connection.
+By default, the command will back up the database from the default DBAL connection.
```
Comment (pounard) [inline docs/content/backup_restore.md:51]: Same.

File: config/db_tools.standalone.complete.sample.yaml (hunk 1 above)
Comment (Lonnytunes) [reply config/db_tools.standalone.complete.sample.yaml:41]: Oh! Indeed! That's a mistake. Thanks!

File: docs/content/backup_restore.md (hunk 4 above)
Comment (Lonnytunes) [reply docs/content/backup_restore.md:39]: Not there. This is the verb we are using here. I believe the contraction is only allowed for the noun.

File: docs/content/configuration/reference.md [hunk 6]
```diff
@@ -62,21 +62,20 @@ class within, then register those as anonymizers.
 ```yaml
 db_tools:
     anonymizer_paths:
-        - '%kernel.project_dir%/vendor/makinacorpus/db-tools-bundle/src/Anonymizer'
-        - '%kernel.project_dir%/src/Anonymization/Anonymizer'
+        - '%kernel.project_dir%/vendor/makinacorpus/db-tools-bundle/src/Anonymization/Anonymizer/Core'
```
[the remark this answers is not in the document]
Comment (Lonnytunes) [reply docs/content/configuration/reference.md:65]: Ok! I will remove the remaining occurrences of this path in documentation and sample files.

File: config/db_tools.standalone.complete.sample.yaml (hunk 2 above)
Comment (Lonnytunes) [reply config/db_tools.standalone.complete.sample.yaml:58]: I checked on Linguee and the singular seems correct.

**Review by SimonMellerin — APPROVED** [review]

**Pull request merged**